brackend/db/models.py

```diff
@@ -97,15 +99,17 @@ class Match(Base):
     round = Column(Integer, ForeignKey("rounds.id"))
     round_position = Column(Integer)
     progress = Column(Enum(MatchProgress), nullable=False)
-    players = relationship("User", secondary="match_users", backref="matches")
+    players = association_proxy("match_users", "user_")
 
 
-class MatchUsers(Base):
+class MatchUser(Base):
     __tablename__ = "match_users"
     id = Column(Integer, primary_key=True)
     match = Column(Integer, ForeignKey("matches.id"))
     user = Column(Integer, ForeignKey("users.id"))
     score = Column(Integer)
+    user_ = relationship("User", backref=backref("match_users", cascade="all, delete-orphan"))
+    match_ = relationship("Match", backref=backref("match_users", cascade="all, delete-orphan"))
```

Comment:
Damn, maybe we should have gone with user_id and match_id originally for the id fields, so we could have user/match for the relationship fields. No biggie since we'll be reading from the Match table and the proxy anyways, but just something to think about for future models

Reply:
lol these were exactly my thoughts